sources/niCURL/src/CURL.cpp

```diff
@@ -1401,6 +1408,98 @@ class cCURL : public cIUnknownImpl<iCURL>
                                       iFetchSink* apSink,
                                       const tStringCVec* apHeaders)
   {
+    if (_hasFetchOverride) {
+      TRACE_FETCH(("niCURL: Using JS fetch override."));
+      cString retString = (const char*) EM_ASM_PTR({
+        console.log("niCURL: Fetching using Module.niCURL.handleFetchOverride");
+        let result = Module.niCURL.handleFetchOverride();
+        return allocateUTF8(result);
+      });
+      if (!retString.empty()) {
```

Comment:
We need a way to indicate that we want the regular fetch because the override will only be for some URL.